src/custom-elements.js

```diff
@@ -35,6 +37,7 @@ if (!priorCustomElements ||
 
   /** @type {!CustomElementRegistry} */
   const customElements = new CustomElementRegistry(internals);
+  customElements.enableCustomizedBuiltins = priorCustomElements.enableCustomizedBuiltins;
```

Comment:
I updated this to work with Closure's `ADVANCED` mode. It was mangling `enableCustomizedBuiltins`, but I fixed that.

This is why the tests were failing in the build. Shouldn't happen anymore